tools/tests/utils-tests.js

```diff
@@ -177,6 +177,7 @@ selftest.define("resume downloads", ['net'], function () {
             ++interruptCount;
             request.emit('error', 'pretend-http-error');
             request.emit('end');
+            request.abort();
```

Comment:
I'm not sure why we didn't need this before, but I was seeing the request continuing to dump data into the stream.